cmd/uhc/cluster/describe/describe.go

```diff
@@ -108,26 +125,76 @@ func run(cmd *cobra.Command, argv []string) {
 	clusterResource := resource.Cluster(argv[0])
 
 	// Retrieve the collection of clusters:
-	response, err := clusterResource.Get().
-		Send()
+	response, err := clusterResource.Get().Send()
+
+	// Get cluster body
+	cluster := response.Body()
+
 	if err != nil {
 		fmt.Fprintf(os.Stderr, "Can't retrieve clusters: %s", err)
 		os.Exit(1)
 	}
 
-	cluster := response.Body()
-	fmt.Printf("ID:       %s\n"+
-		"Name:     %s.%s\n"+
-		"Masters:  %d\n"+
-		"Computes: %d\n"+
-		"Region:   %s\n"+
-		"Multi-az: %t\n",
-		cluster.ID(),
-		cluster.Name(),
-		cluster.DNS().BaseDomain(),
-		cluster.Nodes().Master(),
-		cluster.Nodes().Compute(),
-		cluster.Region().ID(),
-		cluster.MultiAZ(),
-	)
+	if args.output {
+		// Create a filename based on cluster name
```

Comment:
End comments with punctuation, for example: `// Create file name based on cluster name:`